app/javascript/controllers/leaflet_viewer_controller.js

```diff
@@ -155,7 +156,16 @@ export default class LeafletViewerController extends Controller {
       bounds.getSouthEast(),
       bounds.getNorthEast(),
       bounds.getNorthWest(),
-    ]);
+    ],
+      {
```

Comment:
Do we want this for all views? I am assuming we don't. I would do an if statement here. If mapGeomValue have these settings.

Reply:
The additional dashed border causes no harm on simple boundaries, as it overlays exactly. Example below:

![Screenshot 2024-08-09 at 4 30 44 PM](https://github.com/user-attachments/assets/6cd3c91e-9e66-4149-b742-ed743837ce4b)

But, the code would likely be clearer with another conditional... I'll give that a try.

Reply:
I don't think it needs it if it doesn't apply it to the large area, I'm not quite sure how it isn't though. Maybe I am reading it wrong?

Reply:
@dnoneill // I made the polyline options conditional here: https://github.com/geoblacklight/geoblacklight/blob/02c677a91f4f6162076da70ca060ebd21894d049/app/javascript/controllers/leaflet_viewer_controller.js#L160-L166

I think that reads a lot better, but let me know what you think!